src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -42,6 +44,10 @@ trait GraphDAO {
     workspacePipeline(db, workspaceNamespace, workspaceName).out(entityType).filter(hasProperty("_name", entityName))
   }
 
+  def methodConfigPipeline(db: Graph, workspaceNamespace: String, workspaceName: String, methodConfigNamespace: String, methodConfigName: String) = {
+    workspacePipeline(db, workspaceNamespace, workspaceName).out(MethodConfigEdgeType).filter(hasProperties(Map("namespace" -> methodConfigNamespace, "name" -> methodConfigName)))
```

Comment:
We don't need them per se, but for consistency's sake I think it makes more sense (i.e. when you look at a vertex in the graph, it should be immediately clear what properties are user-defined vs not).